src/QuickMarcEditor/QuickMarcEditor.js

```diff
@@ -300,15 +300,11 @@ const QuickMarcEditor = ({
         callNumber: instance?.callNumber,
       };
     } else if ((marcType === MARC_TYPES.AUTHORITY) && records.length) {
-      const currentHeading = records.find((recordRow) => {
-        return recordRow.tag === recordInfoProps.correspondingMarcTag;
-      });
-
       const initialHeading = initialValues.records.find((recordRow) => {
         return recordRow.tag === recordInfoProps.correspondingMarcTag;
       });
 
-      const headingContent = currentHeading?.content || initialHeading?.content;
+      const headingContent = initialHeading?.content;
```

Comment:
I forgot about the case when we're creating records. Let's confirm with Christine what we should show as the title

Reply:
Christine [replied](https://issues.folio.org/browse/UIQM-555?focusedCommentId=200589&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-200589) there should always be `Create a new MARC authority record` pane title.